src/redux/actions/syncBattle.ts

```diff
@@ -165,7 +168,7 @@ export const syncBattle = createAsyncThunk<CalcdexBattleState, SyncBattlePayload
     // determine if we should look for team sheets
     const sheetStepQueues = (
       !!settings?.autoImportTeamSheets
-        && battle.stepQueue?.filter((q) => (q.startsWith('|c|') && q.includes('/raw')) || q.startsWith('|uhtml|ots|'))
+        && battle.stepQueue?.filter((q) => (q.startsWith('|c|') && q.includes('/raw')) || q.startsWith('|uhtml|ots|') || q.includes('|raw|<div class="infobox" style="margin-top:5px">'))
```

Comment:
damn, did they change it to `'|raw|'` instead of `'|uhtml|ots|'` ? :c

thanks!

Reply:
This is only for the "force open team sheet" option which is enforced in the bo3 ladder. Think the other methods of team sheet parsing are still valid for accepting OTS and showteam commands.